requirements-rtd.txt

```diff
@@ -1,4 +1,3 @@
 repoze.sphinx.autointerface
 Zope
 ZConfig==3.5.0
-tempstorage
```

Comment:
This is still required as long as the ZConfig schema contains the `tempstorge` declaration.
See https://readthedocs.org/projects/zope/builds/9537827/ for the failed RTD build.

Reply:
done